_rules/css-restrict-orientation-b33eff.md

```diff
@@ -23,17 +23,20 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible](#visible) and has a CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property that is applied conditionally on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) with a value of `landscape` or `portrait`, where the CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property has any of the below [transformation functions](https://www.w3.org/TR/css-transforms/#transform-functions):
+This rule applies to any [HTML element][] that is [visible](#visible) and has one of the following CSS properties applied conditionally on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) with a value of `landscape` or `portrait`:
+
+1. the CSS [rotate](https://www.w3.org/TR/css-transforms-2/#individual-transforms) property
+2. the CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property with any of the below [transformation functions](https://www.w3.org/TR/css-transforms/#transform-functions):
 
 - [rotate](https://www.w3.org/TR/css-transforms/#funcdef-transform-rotate)
-- [rotate3d](https://drafts.csswg.org/css-transforms-2/#funcdef-rotate3d)
-- [rotateZ](https://drafts.csswg.org/css-transforms-2/#funcdef-rotatez)
+- [rotate3d](https://www.w3.org/TR/css-transforms-2/#funcdef-rotate3d)
+- [rotateZ](https://www.w3.org/TR/css-transforms-2/#funcdef-rotatez)
 - [matrix](https://www.w3.org/TR/css-transforms/#funcdef-transform-matrix)
-- [matrix3d](https://drafts.csswg.org/css-transforms-2/#funcdef-matrix3d)
+- [matrix3d](https://www.w3.org/TR/css-transforms-2/#funcdef-matrix3d)
```

Comment:
These need an extra indentation.

Reply:
> These need an extra indentation.

Indent added and other suggestions committed